packages/roosterjs-editor-types/lib/enum/ExperimentalFeatures.ts

```diff
@@ -144,6 +144,7 @@ export const enum ExperimentalFeatures {
     TabKeyTextFeatures = 'TabKeyTextFeatures',
 
     /**
+     * @deprecated This feature is always enabled
```

Comment:
Please move it up before line 139 to put this feature flag together with other deprecated ones.